_posts/2015-04-30-gsoc-15-is-now-live.markdown

```diff
@@ -0,0 +1,123 @@
+---
+layout: post
+title: Google Summer of Code 2015 is now live
+categories:
+- activism
+author: febuiles
+published: true
+date: 2015-04-30 15:00:00 +00:00
+---
+**TODO** Confirm Josh K. participation and remove **TODO** tags.
+
+Google has announced the list of [accepted projects][melange] for this year's [Google Summer of
+Code][gsoc] (GSoC). Rails has been granted 8 slots, here's a brief introduction to the projects and
+the people behind them.
+
+## Performance Enhancements for the Asset Pipeline
+
+**Student:** [Martha de Luque]()    
```

Comment:
I contacted Martha a couple of days ago but I haven't received a reply yet confirming this is her username. If we don't get an answer I guess we can link to Twitter or remove the <a>.